scarb/src/core/manifest/toml_manifest.rs

```diff
@@ -704,6 +744,56 @@ impl TomlManifest {
     }
 }
 
+/// Returns the absolute canonical path of the README file for a [`TomlPackage`].
+pub fn readme_for_package(
+    package_root: &Utf8Path,
+    readme: Option<&PathOrBool>,
+) -> Result<Option<Utf8PathBuf>> {
+    match readme {
+        None => abs_canonical_path(
+            package_root,
+            default_readme_from_package_root(package_root.parent().unwrap()),
+        ),
+        Some(value) => match value {
+            PathOrBool::Bool(false) => Ok(None),
+            PathOrBool::Bool(true) => abs_canonical_path(package_root, Some("README.md".into())),
```

Comment:
Shouldn't we look for `default_readme_from_package_root` here also? This looks inconsistent to me

Reply:
Now it is accurate with the documentation (manifest.md change) and how cargo does it.
I don't have a strong opinion on this.
Perhaps setting `readme = true` and not setting it at all, should be treated differently, but I can change it and add the check for `readme = true`, that ensures the file exists.